pydoctor/test/test_astbuilder.py

```diff
@@ -1971,3 +1972,32 @@ class j: pass
     assert system.allobjects['top._impl'].resolveName('f') == system.allobjects['top'].contents['f']
     assert system.allobjects['_impl2'].resolveName('i') == system.allobjects['top'].contents['i']
     assert all(n in system.allobjects['top'].contents for n in  ['f', 'g', 'h', 'i', 'j'])
+
+@systemcls_param
+def test_prepend_package(systemcls: Type[model.System]) -> None:
+    system = systemcls()
```

Comment:
Maybe add a docstring to these tests to  explain in simple words what is the purpose of this tests.
What is specific to this tests.

This shold be done to make it clear why we need another test like `test_prepend_package_real_path`